### PR TITLE
fix(react-utilities): Allow `isSlot` to work with multiple utilities instances

### PR DESCRIPTION
**Pull request by daniel-hauser**
## Previous Behavior

Some complex projects might end up with multiple instances (not necessarily from different versions) of `react-utilities`.

In that case, `isSlot` will `erroneously` return false, causing react to throw the following error when rendering the slots.
```
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object. 
```
#29434 added a warning on cases like this, but in large systems it's not always easy to fix.

As described in that PR, this is due to the fact Symbols are unique and having different instances of `react-utilities` running means having multiple Symbols.

## New Behavior

Moving from `Symbol` to [`Symbol.for`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/for) will share the same symbol reference in a runtime-wide registry, fixing the issue and allowing `isSlot` to return the correct answer.

## Related Issue(s)

None

### CODE REVIEW

**Review by ling1726 — APPROVED** [review]

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/Avatar-mount_base.html">656</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/Avatar-mount.html">648</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/Button-mount_base.html">313</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/Button-mount.html">314</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/Field-mount_base.html">1118</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/Field-mount.html">1129</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/FluentProvider-mount_base.html">683</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/FluentProvider-mount.html">711</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/FluentProviderWithTheme-mount_base.html">79</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/FluentProviderWithTheme-mount.html">75</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">56</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">56</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">75</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">74</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/MakeStyles-mount_base.html">851</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/MakeStyles-mount.html">842</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/Persona-mount_base.html">1791</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/Persona-mount.html">1754</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/SpinButton-mount_base.html">1400</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30209/perf-test-react-components/SpinButton-mount.html">1407</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                             |      Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                Change |
| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |   `93.88 kB`<br />`28.673 kB` |  `93.888 kB`<br />`28.673 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |  `83.729 kB`<br />`23.458 kB` |  `83.737 kB`<br />`23.458 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `50.167 kB`<br />`15.945 kB` |  `50.175 kB`<br />`15.944 kB` | `8 B` <br />`-1 B`  |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          |   `19.696 kB`<br />`7.795 kB` |   `19.704 kB`<br />`7.796 kB` |  `8 B` <br />`1 B`  |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                                  |  `64.823 kB`<br />`20.274 kB` |  `64.831 kB`<br />`20.274 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |   `26.897 kB`<br />`8.729 kB` |   `26.905 kB`<br />`8.729 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `27.798 kB`<br />`9.024 kB` |   `27.806 kB`<br />`9.024 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |   `25.303 kB`<br />`9.307 kB` |   `25.311 kB`<br />`9.306 kB` | `8 B` <br />`-1 B`  |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `39.505 kB`<br />`11.17 kB` |  `39.513 kB`<br />`11.169 kB` | `8 B` <br />`-1 B`  |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |  `46.866 kB`<br />`12.662 kB` |  `46.874 kB`<br />`12.661 kB` | `8 B` <br />`-1 B`  |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |  `44.284 kB`<br />`12.544 kB` |  `44.292 kB`<br />`12.542 kB` | `8 B` <br />`-2 B`  |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |  `52.298 kB`<br />`14.134 kB` |  `52.306 kB`<br />`14.134 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |  `56.551 kB`<br />`13.068 kB` |  `56.559 kB`<br />`13.068 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `94.677 kB`<br />`27.247 kB` |  `94.685 kB`<br />`27.248 kB` |  `8 B` <br />`1 B`  |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `89.501 kB`<br />`25.733 kB` |  `89.509 kB`<br />`25.733 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |   `13.056 kB`<br />`5.386 kB` |   `13.064 kB`<br />`5.385 kB` | `8 B` <br />`-1 B`  |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |   `15.299 kB`<br />`6.143 kB` |   `15.307 kB`<br />`6.143 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |   `14.007 kB`<br />`5.752 kB` |   `14.015 kB`<br />`5.752 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-checkbox</samp> <br /> <abbr title='bundle-size/Checkbox.fixture.js'>Checkbox</abbr>                                                                              |  `34.738 kB`<br />`11.903 kB` |  `34.746 kB`<br />`11.903 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 |  `95.049 kB`<br />`30.679 kB` |  `95.057 kB`<br />`30.679 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 |   `93.64 kB`<br />`30.363 kB` |  `93.648 kB`<br />`30.363 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `69.893 kB`<br />`20.261 kB` |  `69.901 kB`<br />`20.261 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `209.769 kB`<br />`59.924 kB` | `209.777 kB`<br />`59.924 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |   `42.38 kB`<br />`14.104 kB` |  `42.388 kB`<br />`14.103 kB` | `8 B` <br />`-1 B`  |
| <samp>react-datepicker-compat</samp> <br /> <abbr title='bundle-size/DatePicker.fixture.js'>DatePicker Compat</abbr>                                                          | `213.529 kB`<br />`59.885 kB` | `213.537 kB`<br />`59.885 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>                                                    |  `95.444 kB`<br />`28.457 kB` |  `95.452 kB`<br />`28.456 kB` | `8 B` <br />`-1 B`  |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |   `20.812 kB`<br />`7.763 kB` |    `20.82 kB`<br />`7.763 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-field</samp> <br /> <abbr title='bundle-size/Field.fixture.js'>Field</abbr>                                                                                       |   `22.419 kB`<br />`8.573 kB` |   `22.427 kB`<br />`8.573 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |   `15.728 kB`<br />`6.229 kB` |   `15.736 kB`<br />`6.228 kB` | `8 B` <br />`-1 B`  |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoButton.fixture.js'>InfoButton</abbr>                                                                        | `132.639 kB`<br />`41.721 kB` | `132.647 kB`<br />`41.722 kB` |  `8 B` <br />`1 B`  |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoLabel.fixture.js'>InfoLabel</abbr>                                                                          | `136.326 kB`<br />`42.862 kB` | `136.334 kB`<br />`42.861 kB` | `8 B` <br />`-1 B`  |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |   `26.879 kB`<br />`9.118 kB` |   `26.887 kB`<br />`9.118 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-jsx-runtime</samp> <br /> <abbr title='bundle-size/Classic.fixture.js'>Classic Pragma</abbr>                                                                      |       `1.049 kB`<br />`528 B` |       `1.057 kB`<br />`530 B` |  `8 B` <br />`2 B`  |
| <samp>react-jsx-runtime</samp> <br /> <abbr title='bundle-size/DevRuntime.fixture.js'>JSX Dev Runtime</abbr>                                                                  |    `3.775 kB`<br />`1.648 kB` |    `3.783 kB`<br />`1.648 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-jsx-runtime</samp> <br /> <abbr title='bundle-size/Runtime.fixture.js'>JSX Runtime</abbr>                                                                         |    `4.369 kB`<br />`1.881 kB` |    `4.377 kB`<br />`1.881 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |   `14.141 kB`<br />`5.759 kB` |   `14.149 kB`<br />`5.759 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `17.074 kB`<br />`6.911 kB` |   `17.082 kB`<br />`6.911 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          |  `141.96 kB`<br />`43.516 kB` | `141.968 kB`<br />`43.516 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `144.646 kB`<br />`44.025 kB` | `144.654 kB`<br />`44.025 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-message-bar</samp> <br /> <abbr title='bundle-size/Default.fixture.js'>MessageBar (all components)</abbr>                                                         |   `25.173 kB`<br />`9.039 kB` |   `25.181 kB`<br />`9.039 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                                                                                 |  `57.058 kB`<br />`17.822 kB` |  `57.066 kB`<br />`17.821 kB` | `8 B` <br />`-1 B`  |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 |  `120.804 kB`<br />`38.11 kB` |  `120.812 kB`<br />`38.11 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/ProgressBar.fixture.js'>ProgressBar</abbr>                                                                        |    `17.42 kB`<br />`6.897 kB` |   `17.428 kB`<br />`6.898 kB` |  `8 B` <br />`1 B`  |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |    `22.92 kB`<br />`8.472 kB` |   `22.928 kB`<br />`8.472 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                                                                       |  `31.738 kB`<br />`10.365 kB` |  `31.746 kB`<br />`10.365 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                                                             |   `15.337 kB`<br />`6.263 kB` |   `15.345 kB`<br />`6.263 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |  `28.601 kB`<br />`10.206 kB` |  `28.609 kB`<br />`10.205 kB` | `8 B` <br />`-1 B`  |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |   `39.142 kB`<br />`12.77 kB` |   `39.15 kB`<br />`12.769 kB` | `8 B` <br />`-1 B`  |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                                                        |  `36.766 kB`<br />`11.789 kB` |  `36.774 kB`<br />`11.788 kB` | `8 B` <br />`-1 B`  |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |   `23.394 kB`<br />`8.467 kB` |   `23.402 kB`<br />`8.467 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |  `34.222 kB`<br />`10.995 kB` |   `34.23 kB`<br />`10.995 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                                 | `154.888 kB`<br />`43.533 kB` | `154.896 kB`<br />`43.535 kB` |  `8 B` <br />`2 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                                            |  `43.891 kB`<br />`13.782 kB` |  `43.899 kB`<br />`13.781 kB` | `8 B` <br />`-1 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                                 | `128.109 kB`<br />`34.792 kB` | `128.117 kB`<br />`34.791 kB` | `8 B` <br />`-1 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                                         |  `73.288 kB`<br />`20.017 kB` |  `73.296 kB`<br />`20.016 kB` | `8 B` <br />`-1 B`  |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                                   |  `71.895 kB`<br />`19.617 kB` |  `71.903 kB`<br />`19.617 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/InteractionTag.fixture.js'>InteractionTag</abbr>                                                                      |   `15.251 kB`<br />`6.058 kB` |   `15.259 kB`<br />`6.058 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/tag.fixture.js'>Tag</abbr>                                                                                            |    `29.974 kB`<br />`9.44 kB` |   `29.982 kB`<br />`9.439 kB` | `8 B` <br />`-1 B`  |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/TagGroup.fixture.js'>TagGroup</abbr>                                                                                  |   `74.323 kB`<br />`22.28 kB` |   `74.331 kB`<br />`22.28 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `16.697 kB`<br />`6.569 kB` |   `16.705 kB`<br />`6.569 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |    `19.87 kB`<br />`6.896 kB` |   `19.878 kB`<br />`6.896 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                                                              |  `30.939 kB`<br />`10.476 kB` |  `30.947 kB`<br />`10.476 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-timepicker-compat</samp> <br /> <abbr title='bundle-size/TimePicker.fixture.js'>TimePicker</abbr>                                                                 |  `96.886 kB`<br />`32.123 kB` |  `96.894 kB`<br />`32.123 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-timepicker-compat-preview</samp> <br /> <abbr title='bundle-size/TimePicker.fixture.js'>TimePicker</abbr>                                                         |  `96.886 kB`<br />`32.123 kB` |  `96.894 kB`<br />`32.123 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-toast</samp> <br /> <abbr title='bundle-size/Toast.fixture.js'>Toast (including Toaster)</abbr>                                                                   |  `93.195 kB`<br />`28.026 kB` |  `93.203 kB`<br />`28.026 kB` |                                                                                                 `8 B` <br /> |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `53.244 kB`<br />`18.807 kB` |  `53.252 kB`<br />`18.806 kB` | `8 B` <br />`-1 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                          |          Size (minified/GZIP) |
| -------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContext.fixture.js'>createContext</abbr>                 |          `510 B`<br />`328 B` |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContextSelector.fixture.js'>createContextSelector</abbr> |          `537 B`<br />`339 B` |
| <samp>react-calendar-compat</samp> <br /> <abbr title='bundle-size/Calendar.fixture.js'>Calendar Compat</abbr>             | `142.162 kB`<br />`37.149 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                            |   `12.837 kB`<br />`4.818 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                 |    `12.788 kB`<br />`4.67 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>      |    `7.099 kB`<br />`2.385 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                 |   `25.767 kB`<br />`9.308 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                    |          `180 B`<br />`160 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/c00619261f115f3df127867abc79e3ec1601476a'>c00619261f115f3df127867abc79e3ec1601476a</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-utilities/src/compose/isSlot.test.tsx [hunk 1]
```diff
@@ -34,4 +34,11 @@ describe('isSlot', () => {
   it('handles actual slot', () => {
     expect(isSlot(slot.optional({}, { elementType: 'div' }))).toEqual(true);
   });
+
+  it('handles slots created with a diffrent instance of react-utilities', async () => {
+    jest.isolateModules(() => {
+      const otherSlot = require('./slot') as typeof slot;
+      expect(isSlot(otherSlot.optional({}, { elementType: 'div' }))).toBeTruthy();
+    })
```
Comment (daniel-hauser) [inline packages/react-components/react-utilities/src/compose/isSlot.test.tsx:42]: Fix formatting

```suggestion
    });
```

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-utilities/src/compose/isSlot.test.tsx [hunk 2]
```diff
@@ -34,4 +34,11 @@ describe('isSlot', () => {
   it('handles actual slot', () => {
     expect(isSlot(slot.optional({}, { elementType: 'div' }))).toEqual(true);
   });
+
+  it('handles slots created with a diffrent instance of react-utilities', async () => {
```
Comment (khmakoto) [inline packages/react-components/react-utilities/src/compose/isSlot.test.tsx:38]:
```suggestion
  it('handles slots created with a different instance of react-utilities', async () => {
```

Comment (daniel-hauser) [reply packages/react-components/react-utilities/src/compose/isSlot.test.tsx:38]: Oops, thanks!

**Review by khmakoto — APPROVED** [review]

**Pull request merged**